app/retail/templates/emails/gdpr_update.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+{% load static %}
+
+{% block content %}
+
+<h1>{% trans "Gitcoin: Updated Terms & Policies" %}</h1>
+<div style="text-align: left! important">
+    <p>
+        {% trans "Hello" %} {{subscriber}},
```

Comment:
Just a note... we should get into the habit of using `{{ var }}` formatted variables in django templates.

Reply:
I know. But since I saw this is the format followed most placed, so I thought of sticking with it to maintain uniformity. I literally changed from `{{ var }}` to `{{var}}` :yum:

Reply:
This is @mbeacom  when me and @SaptakS try to help out :trollface: 

![not-helpful](https://user-images.githubusercontent.com/5358146/40487527-152d9fde-5f82-11e8-9ed2-505fb8cb8c5d.png)

Reply:
😂